### DISCUSSION TITLE
CacheManager 类的 Instance 静态属性可能为 null

### DISCUSSION THREAD

**Discussion by a44281071** [dotnetcore/BootstrapBlazor#3752] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

CacheManager类的Instance是从实例构造里初始化的。
这就会导致在某些情况下，直接访问CacheManager造成null异常。
这通常会在Utility静态类中的一些方法中重现异常，如初始化DataTableDynamicContext实例。

### .NET Version

net8.0

### Anything else?

库版本：8.6.4为止有此类问题。

**Comment by bb-auto** [comment]
@a44281071 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment]
@a44281071 请提供复现工程
